### DISCUSSION TITLE
Backups and auditing changes ?

### DISCUSSION THREAD

**Discussion by geekmush** [semaphoreui/semaphore#1657] [Q&A] [answered]
How do you view or backup the database that all the operational "stuff" gets stored in?  (and restore!)

Also, how do you *audit* changes made through the WebUI???  Seems that is kind of an important feature, right?

**Comment by tboerger** [comment, marked as answer]
Generally it is always good to fully backup the database. Beside that you are also able to export/import projects separately. In terms of auditing there is so far only the events view, nothing else. But there is a pending webhook integration where you should also be able to use some external system to track audit records fed by the webhook system.

If the events view is not powerful enough feel free to provide suggestions how to improve it.
